docs/Makefile

```diff
@@ -175,3 +175,12 @@ pseudoxml:
 	$(SPHINXBUILD) -b pseudoxml $(ALLSPHINXOPTS) $(BUILDDIR)/pseudoxml
 	@echo
 	@echo "Build finished. The pseudo-XML files are in $(BUILDDIR)/pseudoxml."
+
+makedocsmessages: gettext
+	cd docs && sphinx-intl update -p _build/locale -l en
+
+ensurecrowdinclient:
+	ls -l crowdin-cli.jar || wget https://crowdin.com/downloads/crowdin-cli.jar # make sure we have the official crowdin cli client
+
+uploaddocsmessages: ensurecrowdinclient
+	java -jar crowdin-cli.jar upload sources -b `git symbolic-ref HEAD | xargs basename`
```

Comment:
I think, as this is just a way to have our current practice work for the docs separately, I am ok with merging this as is, but I've made an issue for the problems that you bring up: #2007